wrappers/python/simtk/openmm/app/topology.py

```diff
@@ -155,7 +182,7 @@ def addAtom(self, name, element, residue, id=None):
         residue._atoms.append(atom)
         return atom
 
-    def addBond(self, atom1, atom2):
+    def addBond(self, atom1, atom2, type=None, order=None):
```

Comment:
@jchodera - see #1443, and specifically https://github.com/pandegroup/openmm/issues/1443#issuecomment-256101964 . The idea was to allow both as it will provide for easier bookkeeping especially in "in between" cases (Chris Bayly's example was formal single and double bonds around pyridine or furan, but associate with the "aromatic" type associated).

I'm not necessarily averse to adding a check that `type` and `order` are compatible when both specified, but this would require a chemistry engine to handle generally (rather than just catching obvious incompatibilities like the one you just noted), so I think it's out for the near term.